ubuntu1604/cloudbuild.yaml

```diff
@@ -51,15 +58,18 @@ steps:
 # We use container_test rule, which is a Bazel wrapper of container_structure_test.
 # https://github.com/bazelbuild/rules_docker/blob/master/contrib/test.bzl
   - name: "l.gcr.io/google/bazel"
-    args: ["--output_base=/workspace/output_base", "test", "--test_output=errors", "//:image-test",
+    args: ["--output_base=/workspace/output_base", "test",
+           "--test_output=errors",
            "--incompatible_disable_deprecated_attr_params=false",
            "--incompatible_new_actions_api=false",
            "--incompatible_no_support_tools_in_action_inputs=false",
-           "--host_force_python=PY2"]
+           "--host_force_python=PY2",
```

Comment:
Is it needed here even though it's specified in the .bazelrc file?
Similarly for ubuntu1804